### ISSUE TITLE
jenkins 2.0 workflow/pipeline support

### ISSUE THREAD

**Issue by elliotirwinp4l** [janinko/ghprb#409]
Will you be adding support for pipeline job type?

**Comment by chrislovecnm** [comment]
Can someone outline what needs to happen to get this done?  Can we script this in dsl?

**Comment by bluesliverx** [comment]
See this readme for how to add pipeline DSL support: https://github.com/jenkinsci/pipeline-plugin/blob/master/DEVGUIDE.md

**Comment by boonware** [comment]
Any update on this? It would be great to be able to trigger pipelines without the need for https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Organization+Folder+Plugin

**Comment by jasonkuster** [comment]
FYI, #497 is in progress for this issue. #479 is also a dupe of this issue.

**Comment by benpatterson** [comment]
yes folks, the other repo is officially the parent (that was only reflected in github recently). Working on it there though I don't have a firm timeline on it.
